internal/deviceplugin/server.go

```diff
@@ -125,7 +127,10 @@ func (srv *server) Allocate(ctx context.Context, rqt *pluginapi.AllocateRequest)
 				if cresp.Envs == nil {
 					cresp.Envs = make(map[string]string)
 				}
-				cresp.Envs[key] = value
+				re := regexp.MustCompile(`[0-9]+`)
```

Comment:
Better to compile the regexp only once as a module variable. Here we do the same calculation on every function call and every loop iteration.

Reply:
Good catch! Will definitely change this.